gp-easy-passthrough/gpep-edit-entry.php

```diff
@@ -241,11 +244,35 @@ public function exclude_edit_entry_from_inventory( $query, $field ) {
 		return $query;
 	}
 
+	public function process_feeds( $entry, $form ) {
+		if ( ! $this->process_feeds ) {
+			return $entry;
+		}
+
+		/**
+		 * Disable asynchronous feed process on edit otherwise async feeds will not be re-ran due to a check in
+		 * class-gf-feed-processor.php that checks `gform_get_meta( $entry_id, 'processed_feeds' )` and there isn't
+		 * a way to bypass it.
+		 */
+		$filter_priority = rand( 100000, 999999 );
+		add_filter( 'gform_is_feed_asynchronous', '__return_false', $filter_priority );
+
+		foreach ( GFAddOn::get_registered_addons( true ) as $addon ) {
+			if ( method_exists( $addon, 'maybe_process_feed' ) && ( $this->process_feeds === true || strpos( $this->process_feeds, $addon->get_slug() ) !== false ) ) {
```

Comment:
I like it!

If `process_feeds` is true, should it still skip GPEP like you previously had or does it not cause any issues if it runs?

Reply:
@claygriffiths I ommitted the exception I had for GPEP, since I noticed no anomalies. It does make sense though to explicitly add that as an exception, but I have no proof to claim that it was causing any issues on any of the tests I conducted.